tasks.yaml

```diff
@@ -54,6 +54,42 @@ tasks:
         setVariables:
           - name: BUILD_ARGS
 
+  - name: download-tofu-mac-intel
+    description: download tofu for go:embed
+    actions:
+      - cmd: mkdir -p src/cmd/bin
+      - cmd: rm -f src/cmd/bin/tofu
+      - cmd: wget https://github.com/opentofu/opentofu/releases/download/v1.9.0/tofu_1.9.0_darwin_amd64.zip -O src/cmd/bin/tofu_1.9.0_darwin_amd64.zip
+      - cmd: unzip -p src/cmd/bin/tofu_1.9.0_darwin_amd64.zip tofu > src/cmd/bin/tofu
+      - cmd: rm src/cmd/bin/tofu_1.9.0_darwin_amd64.zip
+
+  - name: download-tofu-mac-apple
+    description: download tofu for go:embed
+    actions:
+      - cmd: mkdir -p src/cmd/bin
+      - cmd: rm -f src/cmd/bin/tofu
+      - cmd: wget https://github.com/opentofu/opentofu/releases/download/v1.9.0/tofu_1.9.0_darwin_arm64.zip -O src/cmd/bin/tofu_1.9.0_darwin_arm64.zip
+      - cmd: unzip -p src/cmd/bin/tofu_1.9.0_darwin_arm64.zip tofu > src/cmd/bin/tofu
+      - cmd: rm src/cmd/bin/tofu_1.9.0_darwin_arm64.zip
+
+  - name: download-tofu-linux-arm
+    description: download tofu for go:embed
+    actions:
+      - cmd: mkdir -p src/cmd/bin
+      - cmd: rm -f src/cmd/bin/tofu
+      - cmd: wget https://github.com/opentofu/opentofu/releases/download/v1.9.0/tofu_1.9.0_linux_arm64.zip -O src/cmd/bin/tofu_1.9.0_linux_arm64.zip
+      - cmd: unzip -p src/cmd/bin/tofu_1.9.0_linux_arm64.zip tofu > src/cmd/bin/tofu
+      - cmd: rm src/cmd/bin/tofu_1.9.0_linux_arm64.zip
+
+  - name: download-tofu-linux-amd
+    description: download tofu for go:embed
+    actions:
+      - cmd: mkdir -p src/cmd/bin
+      - cmd: rm -f src/cmd/bin/tofu
+      - cmd: wget https://github.com/opentofu/opentofu/releases/download/v1.9.0/tofu_1.9.0_linux_amd64.zip -O src/cmd/bin/tofu_1.9.0_linux_amd64.zip
+      - cmd: unzip -p src/cmd/bin/tofu_1.9.0_linux_amd64.zip tofu > src/cmd/bin/tofu
+      - cmd: rm src/cmd/bin/tofu_1.9.0_linux_amd64.zip
```

Comment:
Now that I know this works, all of this could be variableized more. However, given that I hope we move over to using a Make styled build system, I'm not too inclined to really spend the time to spend more time with this until we finis out more of our POC.